themes/robbyrussell/robbyrussell.theme.bash

```diff
@@ -13,13 +13,16 @@ GIT_THEME_PROMPT_SUFFIX="${bold_blue})"
 RVM_THEME_PROMPT_PREFIX="|"
 RVM_THEME_PROMPT_SUFFIX="|"
 
+VIRTUALENV_THEME_PROMPT_PREFIX='('
+VIRTUALENV_THEME_PROMPT_SUFFIX=') '
+
 function git_prompt_info() {
 	git_prompt_vars
 	echo -e "$SCM_PREFIX${bold_red}$SCM_BRANCH$SCM_STATE$SCM_SUFFIX"
 }
 
 function prompt_command() {
-	PS1="${bold_green}➜  ${bold_cyan}\W${reset_color}$(scm_prompt_info)${normal} "
+  PS1="$(conda_or_venv_prompt)${bold_green}➜  ${bold_cyan}\W${reset_color}$(scm_prompt_info)${normal} "
```

Comment:
This also changes indentation.

Reply:
#2302 Ah, OK. Never mind.

Reply:
Hmm, sorry. This is not fixed by #2302. I've mistaken this as #2065.

Reply:
Adding this to a general fix PR and marking the file clean. thanks for spotting!